samples/winapi_drivelist/main.c

```diff
@@ -9,45 +9,47 @@ int main(void)
 
     // Mount some drives for demonstration purposes
     BOOL ret;
+    DWORD error;
     ret = nxMountDrive('C', "\\Device\\Harddisk0\\Partition2\\");
     if (!ret) {
-        debugPrint("Failed to mount C: drive!\n");
+        // Additional error info can be retrieved with GetLastError()
+        error = GetLastError();
+        debugPrint("Failed to mount C: drive! Error code: %x\n", error);
         Sleep(5000);
         return 1;
```

Comment:
This return value is now different compared to the other handling handling. I suggest to remove these two lines and replace them with a goto to the `Sleep()`-loop (you can name the label `end` or something).